src/Identicon/Generator/BaseImagePngGenerator.php

```diff
@@ -0,0 +1,16 @@
+<?php
+
+namespace Identicon\Generator;
+
+use Identicon\Generator\GeneratorInterface;
+
+/**
+ * @author Grummfy <grummfy@gmail.com>
+ */
+class BaseImagePngGenerator extends BaseGenerator
+{
+	public function getMimeType()
+	{
+		return 'image/png';
+	}
+}
```

Comment:
Maybe we should add the mime type inside each generator (Gd, Imagick, Svg). This will avoid an other "Base" method.